website/docs/setup/azure-pipelines.md

```diff
@@ -19,7 +19,7 @@ Type: "Inline"
 
 Script:
 ```bash
-curl -L https://github.com/opctl/opctl/releases/download/0.1.47/opctl0.1.47.linux.tgz | sudo tar -xzv -C /usr/local/bin
+curl -L https://github.com/opctl/opctl/releases/download/0.1.48/opctl0.1.48.linux.tgz | sudo tar -xzv -C /usr/local/bin
```

Comment:
just a thought I've had. If you avoided encoding the version name in the tgz filename, this could be replaced with `https://github.com/opctl/opctl/releases/download/latest/opctl.linux.tgz` and wouldn't need to be updated with each version bump.

Reply:
that's a great idea